stubs/Tideways.php

```diff
@@ -69,6 +75,9 @@ private function __construct() {}
 
         public function getId(): string {}
 
+        /**
+         * @param array<string,string> $annotations
```

Comment:
this allows `array<string,int|string|bool>`

Reply:
Fixed.